src/Sulu/Bundle/RouteBundle/Generator/GeneratedRoute.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\RouteBundle\Generator;
+
+use Sulu\Bundle\RouteBundle\Model\RoutableInterface;
+
+/**
+ * Represents a newly generated route. It contains additional the used entity and class to generate the route.
+ */
+class GeneratedRoute
```

Comment:
The generator has to return more information beside the route (class for which it was generated for example. we could use the `Route` entity but that would make it complicated to reuse the code without saving the route.

Maybe only the name is missleading? Maybe `RouteInformation` or something like that?